### ISSUE TITLE
Why so low MOTA on MOT16 train dataset

### ISSUE THREAD

**Issue by kinginsky** [nwojke/deep_sort#11]
Thanks for your wonderful code ~But when I run the code on the 7 train sequences from MOT16 ,I got an MOTA of 17.9.%.The detail of result is as followed. But on the mot16 results page, the deep_sort get a MOTA of  35.3%. I  donnot know the so big difference between each other. The detection we used is the public detection. I try to upload the result on the test sequences of MOT16 in the server,but I have not made it.Thanks for your reply.
********************* Your Benchmark Results (2D) ***********************
 IDF1  IDP  IDR| Rcll  Prcn   FAR|   GT  MT   PT   ML|    FP    FN   IDs    FM|  MOTA  MOTP MOTAL 
 17.8 32.2 12.3| 43.9  63.4  5.26|  517  65  232  220| 27964 61901   729  1680|  17.9  76.9  18.6

**Comment by nwojke** [comment]
The big difference is probably due to using the default/public detections. First, a key idea in SORT is to keep the tracking framework simple and, instead, to spent computational resources outside of the tracker. As such, the method relies on good quality detections.

Second, you might want to adapt some of the parameters if you change detections. In particular, you will have to change the detector confidence threshold `min_confidence`. Depending on the quality of detections, parameters of the track initialization logic might benefit from a tweak as well.

**Comment by kinginsky** [comment]
Thanks, I use the public detection offered by the MOT16 challange dataset. But I run your code using the POI detection results, I get a MOTA 59% at the training dataset,similar to your figure 61% in your paper which is run on the test dataset. Maybe the results with 35.3% is not using the public detection although it marks with public detectors.
    But a new question comes that your method has a advantage of reducing the IDswitchs. But using my own detection, the MOTA of sort is better than the deep_sort. The min_confidence of both are same.Thanks for reply.

**Comment by nwojke** [comment]
I didn't quite understand your first point about the detections and MOT16 scores, but it sounds as if that issue is resolved.

For the second part: The MOTA score is a summary descriptor that accumulates false positive, false negatives, and ID switches. I'd argue that with all of these errors taken into account equally, you can achieve very reasonable MOTA scores at relatively high number of identity switches. SORT does just fine in that regard and the evaluation in the DeepSORT paper also suggests only a small increase in MOTA score at a good decrease in ID switches.

Deep SORT keep tracks alive for multiple frames and, as a consequence, the method is more prone to picking up a false alarm (see the higher number of false positives in Table 2 of the paper). If this is an issue in your application, try increasing the detector confidence threshold `min_confidence` or increase the number of hits for track confirmation via `n_init` [here](https://github.com/nwojke/deep_sort/blob/master/deep_sort/tracker.py#L40). You might be able to balance this towards higher MOTA scores in this way (another simple alternative is to reduce the maximum track age `max_age` in the same line).

**Comment by kinginsky** [comment]
Thanks. I set the same value , like  max_age=1, and n_init =3. At this condiction, do you recognize how bigger the influence of your CNN matching on the tracking results?

**Comment by Phoebe-star** [comment]
how to  evaluate  MOTA and MOTP  value
https://motchallenge.net/data/MOT16/
><
